### ISSUE TITLE
Unable to perform assignment because the size of the left side is 97-by-162-by-40 and the size of the right side is 283-by-368-by-40.

### ISSUE THREAD

**Issue by EvandroMolinari1998** [cerr/CERR#765]
When I do the whole process and use 'radiomicsFeatS = calcGlobalRadiomicsFeatures(scanNum, structNum, paramS, planC);' to compute the features I get the error:

Unable to perform assignment because the size of the left side is 97-by-162-by-40 and the size of
the right side is 283-by-368-by-40.

**Comment by adityaapte** [comment]
Is the structure number passed to this routine assigned to scanNum. If not,
you can use the following to copy structure from one scan to another:

planC = copyStrToScan(structNum,scanNum,planC);

**Comment by EvandroMolinari1998** [comment]
Yes, the struct number is assigned to scanNum. But this error is related to the mask, do I have to match the array of the mask with the array of the dose matrix?

Error in maskToCERRStructure (line 101)
            tmpM(:,:,i) = logical(maskM(:,:,lB));
Error in copyStrToScan (line 100)
planC = maskToCERRStructure(maskM, 1, scanNum, strName, planC);
